### ISSUE TITLE
Add additional scope to Spotify token request

### ISSUE THREAD

**Issue by NonkelTinken** [ejurgensen/forked-daapd#1023]
Hi,

Love all the good work you guys are doing!

I was wondering if it would be possible to include the 'user-read-recently-played' scope in the Spotify token request?

I am requesting this because I would like to get 50 last played songs by using the token request service of forked daapd. Of course since this is unrelated to the app, I understand that this is not your primary concern. Also not aware of the impact of the above.

Many thanks in advance!

**Comment by ejurgensen** [comment]
I'm not sure I understand how you think it should be used? Is to include these songs in the web ui's Recently played view?

**Comment by NonkelTinken** [comment]
I am using a custom UI that I have created. I would like to include a function to load the last 50 spotify songs in the queue and use the service from forked daapd to get a token to call the spotify api.

This is why I included: "Of course since this is unrelated to the app, I understand that this is not your primary concern. Also not aware of the impact of the above" 

So just checking if there is a possibility.....

Many thanks
